cloudtrail/lib/botocore/data/dms/2016-01-01/examples-1.json

```diff
@@ -0,0 +1,1074 @@
+{
+  "version": "1.0",
+  "examples": {
+    "AddTagsToResource": [
+      {
+        "input": {
+          "ResourceArn": "arn:aws:dms:us-east-1:123456789012:endpoint:ASXWXJZLNWNT5HTWCGV2BUJQ7E",
+          "Tags": [
+            {
+              "Key": "Acount",
+              "Value": "1633456"
+            }
+          ]
+        },
+        "output": {
+        },
+        "comments": {
+          "input": {
+            "ResourceArn": "Required. Use the ARN of the resource you want to tag.",
+            "Tags": "Required. Use the Key/Value pair format."
+          },
+          "output": {
+          }
+        },
+        "description": "Adds metadata tags to an AWS DMS resource, including replication instance, endpoint, security group, and migration task. These tags can also be used with cost allocation reporting to track cost associated with AWS DMS resources, or used in a Condition statement in an IAM policy for AWS DMS.",
+        "id": "add-tags-to-resource-1481744141435",
+        "title": "Add tags to resource"
+      }
+    ],
+    "CreateEndpoint": [
+      {
+        "input": {
+          "CertificateArn": "",
+          "DatabaseName": "testdb",
+          "EndpointIdentifier": "test-endpoint-1",
+          "EndpointType": "source",
+          "EngineName": "mysql",
+          "ExtraConnectionAttributes": "",
+          "KmsKeyId": "arn:aws:kms:us-east-1:123456789012:key/4c1731d6-5435-ed4d-be13-d53411a7cfbd",
+          "Password": "pasword",
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 398b7bcb-9718-4ce8-9990-30404f269d73 -->